app/views/stacks/show.html.erb

```diff
@@ -17,6 +17,21 @@
   </div>
 <% end %>
 
+<% if @stack.inaccessible? %>
+  <div class="banner banner--red">
+    <div class="banner__inner wrapper">
+      <div class="banner__content">
+        <h2 class="banner__title">This repository seems to be missing!</h2>
+        <p class="banner__text">
+          Since <%= time_ago_in_words(@stack.inaccessible_since) %>
+          <%= link_to @stack.github_repo_name, github_repo_url(@stack.repo_owner, @stack.repo_name) %> is inaccessible.
+          This could be a permission issue, or the repository have been deleted on GitHub.
+        </p>
+      </div>
+    </div>
+  </div>
+<% end %>
```

Comment:
> **repo** has been inaccessible for 8 days. This could either be due to a permission issue or because the repository has been removed from GitHub.

Might be better copy